static/js/components/search/CustomSortingColumnHeaders_test.js

```diff
@@ -1,35 +1,49 @@
+/* global SETTINGS: false */
 import React from 'react';
-import { shallow } from 'enzyme';
+import { Provider } from 'react-redux';
+import { mount } from 'enzyme';
 import { assert } from 'chai';
 import sinon from 'sinon';
 
+import { PROGRAMS } from '../../test_constants';
+import {
+  receiveGetProgramEnrollmentsSuccess,
+  setCurrentProgramEnrollment,
+} from '../../actions/programs';
 import CustomSortingColumnHeaders from './CustomSortingColumnHeaders';
 import { sortOptions } from '../../components/LearnerSearch';
+import IntegrationTestHelper from '../../util/integration_test_helper';
 
 describe('CustomSortingSelect', () => {
-  let sandbox, setItemsStub;
+  let sandbox, setItemsStub, helper;
 
   beforeEach(() => {
     sandbox = sinon.sandbox.create();
     setItemsStub = sandbox.stub();
+    helper = new IntegrationTestHelper();
+    helper.store.dispatch(receiveGetProgramEnrollmentsSuccess(PROGRAMS));
+    helper.store.dispatch(setCurrentProgramEnrollment(PROGRAMS[0]));
```

Comment:
This action should be unnecessary, the current enrollment is set automatically after the enrollments is received if there is no current enrollment already